pravega/schema-registry
# Issue 209: Update Schema Registry to consume latest Pravega Client

**Change log description**  
In latest Pravega Client few API's have been changed which needs to be handled in Schema Registry

**Purpose of the change**  
Fixes #209 

**What the code does**  
Consumes the latest Pravega Client(0.10) by updating the Client API's

**How to verify it**  
Import Pravega Version should be 0.10 (Snapshot) and all existing test cases should pass with a green build scan
